eplaunch/workflows/manager.py

```diff
@@ -0,0 +1,44 @@
+# import importlib
+from importlib import util as import_util
+import inspect
+import os
+import re
+
+
+def get_workflows(workflow_path=None):
+    this_file_directory_path = os.path.dirname(os.path.realpath(__file__))
+    built_in_workflow_directory = os.path.join(this_file_directory_path, 'builtin', 'workflows')
+
+    if not workflow_path:
+        workflow_path = built_in_workflow_directory
+
+    py_search_regex = re.compile('.py$', re.IGNORECASE)
+    workflow_files = filter(py_search_regex.search, os.listdir(workflow_path))
+
+    modules = []
+    for this_file in workflow_files:
+        if '__init__.py' in this_file:
+            continue
+        this_file_path = os.path.join(workflow_path, this_file)
+        module_spec = import_util.spec_from_file_location('workflow_module', this_file_path)
+        module = import_util.module_from_spec(module_spec)
+        modules.append(module)
+        module_spec.loader.exec_module(module)
```

Comment:
Then we get a ModuleSpec instance from the file path, create a new module from that specification, and append the module to the list.  We then execute the module in the current namespace (doesn't matter though) to initialize it.